src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine.cs

```diff
@@ -78,33 +77,18 @@ private async Task ProcessAsync(ProjectToDocumentMap projectToDocumentMap)
                     return;
                 }
 
-                // Get the connected components of the dependency graph and process each individually.
-                // That way once a component is done we can throw away all the memory associated with
-                // it.
-                // For each connected component, we'll process the individual projects from bottom to
-                // top.  i.e. we'll first process the projects with no dependencies.  Then the projects
-                // that depend on those projects, and so on.  This way we always have created the 
-                // dependent compilations when they're needed by later projects.  If we went the other
-                // way (i.e. processed the projects with lots of project dependencies first), then we'd
-                // have to create all their dependent compilations in order to get their compilation.
-                // This would be very expensive and would take a lot of time before we got our first
-                // result.
-                var connectedProjects = _dependencyGraph.GetDependencySets(_cancellationToken);
```

Comment:
this is no longer an issue as we can still get results uickly by processing hte compilations in parallel, thus returning results quickly for the compilations we can get quickly.